9.6/docker-entrypoint.sh

```diff
@@ -85,8 +85,9 @@ if [ "$1" = 'postgres' ]; then
 			pass=
 			authMethod=trust
 		fi
-
-		{ echo; echo "host all all all $authMethod"; } | tee -a "$PGDATA/pg_hba.conf" > /dev/null
+		
+		echo "" >> $PGDATA/pg_hba.conf
+		echo "host all all all $authMethod" >> $PGDATA/pg_hba.conf
```

Comment:
Ah, the `| tee ...` was a remnant from when we actually had to do `| gosu postgres tee ...`.

IMO, we should instead add some whitespace and use `>>` now, ala:
```bash
	{
		echo
		echo "host all all all $authMethod"
	} >> "$PGDATA/pg_hba.conf"
```